tests/std/tests/P0429R9_flat_map/test.cpp

```diff
@@ -180,13 +237,18 @@ void test_construction() {
         assert(check_requirements(fmap));
         assert(check_key_content(fmap, {0, 1, 2, 3, 38, 242}));
         assert(check_value_content(fmap, {44, 2324, 635462, 433, 5, 7}));
+        static_assert(!is_constructible_v<flat_multimap<int, int>, sorted_unique_t, vector<int>, vector<int>>);

```

Comment:
We should test `sorted_equivalent_t`. There are actually a lot of constructors not being tested here, so I'll add a FIXME comment.